docker-compose.yml

```diff
@@ -4,36 +4,78 @@ services:
     image: postgres:9.3
     container_name: api-java-postgres
 
-  possum:
-    image: registry.tld/possum:0.1.0-stable
-    container_name: api-java-possum
+  conjur:
+    image: cyberark/conjur:0.1.0-stable
+    container_name: api-java-conjur
```

Comment:
can we not set `container_name`? it makes it harder to parallelize and doesn't improve on the `docker-compose` workflow since you can already target `conjur` by name.

Reply:
Good point, I will delete it